modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc

```diff
@@ -25,11 +25,16 @@ the *Filter by name* to find the Operator you want. Then, click on it.
 
 . On the right-hand side of the *Operator Details* page, select *Uninstall
 Operator* from the *Actions* drop-down menu.
-
-. When prompted by the *Remove Operator Subscription* window, you can select the
-*Also completely remove the Operator from the selected namespace* check box to
-remove all components related to the Operator. This removes the CSV, which
-in turn removes the Pods, Deployments, CRDs, and CRs associated with the
-Operator. Leaving this unchecked results in only the Subscription being removed.
-
-. Select *Remove*. This Operator will stop running and no longer receive updates.
++
+An *Uninstall Operator?* dialog box will pop up, reminding you that *Removing
+the operator will not remove any of its custom resource definitions or managed
+resources. If your operator has deployed applications on the cluster or
+configured off-cluster resources, these will continue to run and need to be
+cleaned up manually.*
++
+To reiterate, the Operator, any Operator deployments, and Pods are removed by
+this action. Any resources managed by the Operator, including CRDs and CRs are
+not removed. Also, the web console will show UIs for any CRs unless the CRD is
+manually removed.
```

Comment:
```
Also, the web console will show UIs for any CRs unless the CRD is manually removed
```
What does this sentence mean?

Reply:
@spadgett Should I remove this last line?

Reply:
@yapei This is meant to address special UIs like Chargeback or Severless that aren't disabled when uninstalling the operator since the CRD still exists.

I'd suggest, "The web console enables dashboards and navigation items for some operators. In order to remove these after uninstalling the operator, you might need to manually delete the operator CRDs."

Reply:
Updated. Thanks @spadgett